content/en/guides/core/automations/view-automation-history.md

```diff
@@ -0,0 +1,146 @@
+{{% /pageinfo %}}
+
+This page describes how to view and understand the execution history of your W&B [automations]({{< relref "/guides/core/automations/">}}). The automation history feature allows you to track when automations were triggered, what actions were taken, and whether they succeeded or failed.
+
+## Overview
+
+Automation history provides a comprehensive log of all automation executions, including:
+- **Execution timestamp**: When the automation was triggered
+- **Trigger event**: The specific event that caused the automation to run
+- **Status**: Whether the automation succeeded, failed, or is still running
+- **Action details**: Information about what action was performed (Slack notification sent, webhook called, etc.)
+- **Error messages**: Detailed error information if the automation failed
+
+## Access automation history
+
+You can view automation history from multiple locations:
+
+### From the Automations tab
+
+{{< tabpane text=true >}}
+{{% tab "Registry" %}}
+1. Navigate to your registry and click the **Automations** tab
+2. Find the automation you want to investigate
+3. Click on the automation name to view its details
+4. Select the **History** tab to see all past executions
+
+{{% /tab %}}
+{{% tab "Project" %}}
+1. Navigate to your project and click the **Automations** tab
+2. Find the automation you want to investigate
+3. Click on the automation name to view its details
+4. Select the **History** tab to see all past executions
+
+{{% /tab %}}
+{{< /tabpane >}}
+
+### From the automation details page
+
+When viewing an individual automation:
+1. The **History** tab displays a chronological list of all executions
+2. Each entry shows:
+   - Execution date and time
+   - Triggering event details
+   - Status indicator (success, failure, or in progress)
+   - Duration of execution
+
+## Understanding execution details
+
+Click on any execution entry to view detailed information:
+
+### Successful executions
+For successful executions, you'll see:
+- The exact trigger that initiated the automation
+- The payload or message that was sent
+- Confirmation of successful delivery
+- Response details (for webhook automations)
+
+### Failed executions
+For failed executions, the details include:
+- The error message explaining why the automation failed
+- The stage at which the failure occurred
+- Debugging information to help resolve the issue
+- Options to retry the automation (if applicable)
+
+## Filter and search history
+
+The automation history interface provides several ways to find specific executions:
+
+### Filter by status
+- **All**: View all executions
+- **Successful**: Show only successful executions
+- **Failed**: Show only failed executions
+- **In Progress**: Show currently running automations
+
+### Filter by date range
+- Use the date picker to select a specific time period
+- Options include: Last 24 hours, Last 7 days, Last 30 days, or custom range
```

Comment:
The first sentence is not a list item.
The second sentence is not itself a list item. Render the options as a list.